mem/buffers_test.go

```diff
@@ -0,0 +1,44 @@
+package mem
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestSplit(t *testing.T) {
```

Comment:
These tests should be methods on the `grpctest.Tester` type. Otherwise, our vet will fail.